pkg/commands/login.go

```diff
@@ -7,23 +7,84 @@ package commands
 import (
 	"bufio"
 	"fmt"
+	"io/ioutil"
 	"os"
+	"path/filepath"
+	"strconv"
 	"strings"
 
 	"github.com/scaleway/scaleway-cli/vendor/github.com/Sirupsen/logrus"
 	"github.com/scaleway/scaleway-cli/vendor/golang.org/x/crypto/ssh/terminal"
 
 	"github.com/scaleway/scaleway-cli/pkg/api"
+	"github.com/scaleway/scaleway-cli/pkg/utils"
 )
 
 // LoginArgs are arguments passed to `RunLogin`
 type LoginArgs struct {
 	Organization string
 	Token        string
+	IndexSSH     string
+}
+
+// selectKey allows to choice a key in ~/.ssh
+func selectKey(args *LoginArgs) error {
+	fmt.Println("Do you want to upload a SSH key ?")
+	home, err := utils.GetHomeDir()
+	if err != nil {
+		return err
+	}
+	dir := filepath.Join(home, ".ssh")
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
+		return fmt.Errorf("Unable to open your ~/.ssh: %v", err)
+	}
+	var pubs []string
+
+	for i := range files {
+		if filepath.Ext(files[i].Name()) == ".pub" {
+			pubs = append(pubs, files[i].Name())
+		}
+	}
+	if len(pubs) == 0 {
+		return nil
+	}
+	fmt.Println("[0] I don't want to upload a key !")
+	for i := range pubs {
+		fmt.Printf("[%d] %s\n", i+1, pubs[i])
+	}
+	for {
+		promptUser("Which [id]: ", &args.IndexSSH, true)
+		id, err := strconv.ParseUint(strings.TrimSpace(args.IndexSSH), 10, 32)
+		if err != nil {
+			fmt.Println(err)
+			continue
+		}
+		if int(id) > len(pubs) {
+			fmt.Println("Out of range id must be lower than", len(pubs))
+			continue
+		}
+		args.IndexSSH = ""
+		if id == 0 {
+			break
+		}
+		buff, err := ioutil.ReadFile(filepath.Join(dir, pubs[id-1]))
+		if err != nil {
+			return fmt.Errorf("Unable to open your key: %v", err)
+		}
+		args.IndexSSH = string(buff[:len(buff)])
+		break
+	}
+	return nil
 }
 
 // RunLogin is the handler for 'scw login'
 func RunLogin(ctx CommandContext, args LoginArgs) error {
+	if args.IndexSSH == "" {
+		if err := selectKey(&args); err != nil {
```

Comment:
You should ask for user SSH key after calling `apiConnection.checkCredentials()`